cmd/starter/main.go

```diff
@@ -74,11 +75,18 @@ func waitForBuild(conn *beanstalk.Conn) {
 			panic(err)
 		}
 
-		repo := string(body)
+		job, err := buildjob.FromBytes(body)
+		if err != nil {
+			// there was a job we could not parse throw it away and continue
+			_ = conn.Delete(id)
+			continue
+		}
+		repo := job.Repository
 
 		tmpDir, err := ioutil.TempDir("", "gobuild")
 		orFail(err)
```

Comment:
Currently out-of-scope but a good catch. As I'm planning to make the workers run in go routines instead of one main thread I'll refactor that in that scope as it is required in there.